src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -873,6 +873,8 @@ internal ImmutableArray<RefKind> ParameterRefKinds
             get;
         }
 
+        internal virtual ImmutableArray<INamedTypeSymbol> CallingConventionTypes => ImmutableArray<INamedTypeSymbol>.Empty;
```

Comment:
>internal virtual ImmutableArray<INamedTypeSymbol> CallingConventionTypes => ImmutableArray<INamedTypeSymbol>.Empty [](start = 8, length = 114)

Adding virtual APIs to symbols is error prone. I guess, it would be fine to to do that if this API was designed to return a non-empty set only for a function pointer signature, which is not subject to generic construction, retargeting and other forms of wrapping. Otherwise, this API should be abstract, should be implemented and tested for most derived types.